refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+- `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+- `TRANSPARENCY.KNOWN` Demands: recommend status = `GRANTED`, and data = `NO`.
+- For any other action : recommend status = `DENIED`, motive=`IDENTITY-UNCONFIRMED`
+
+When Data Subject ID is provided, the Data Subject is known by the System and authenticated:
+- Regardless of restrictions
+    - `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+    - `TRANSPARENCY.KNOWN` Demands: recommend status = `GRANTED`, and data = `YES`.
+    - `TRANSPARENCY.DATA-CATEGORIES` Demands: recommend status = `GRANTED`, and data = list of Data Categories that are included in any of the [Privacy Scope Triples](#privacy-scope-triples) included in the Eligible Privacy Scope.
+    - `TRANSPARENCY.PROVENANCE`, see [Resolving provenance in requests](#resolving-provenance-in-requests)
+    - `TRANSPARENCY.ORGANIZATION`, `TRANSPARENCY.POLICY`, `TRANSPARENCY.RETENTION`, `TRANSPARENCY.DPO`, `TRANSPARENCY.WHERE`, `TRANSPARENCY.WHO`, Demands: recommend status = `GRANTED`, and data = information corresponding to the request taken from configuration settings as defined under [Implications for Systems](./RFC-PRIV.md#design-implications-for-systems-implementing-PRIV) and under [Configuration and Prerequisites](#configuration-and-prerequisites).
+
+- With regards to Demand restrictions (if any) limiting the scope of the Demand:
+    - first, check for presence of incompatible restrictions (and if incompatible recommend status = `DENIED`, motive = `REQUEST-UNSUPPORTED`):
+        - [Consent Restriction](./RFC-PRIV.md#consent-restriction) with any other type of Restriction,
+        - [Consent Restriction](./RFC-PRIV.md#consent-restriction) within a Demand other than `REVOKE-CONSENT`,
+        - More than one [Capture Restriction](./RFC-PRIV.md#capture-restriction),
+        - More than one [Data Reference Restriction](./RFC-PRIV.md#data-reference-restriction)
+        - A Capture Restriction and any other restriction that is not a Privacy Scope Restriction, or a Data Reference Restriction,
+        - More than one [Date Range](./RFC-PRIV.md#date-range) Restriction.
+
+    - second, Calculate **Restriction Scope** and **Concerned Fragments**. This is done by processing every Restriction according to the following approach:
+        - at the beginning set **Restriction Scope** to be equal to the Eligible Privacy Scope of the Data Subject
+        **Concerned Fragments** is set to all Data Capture Fragments the `scope`s of which are included in the **Restriction Scope**.
+
+        - when processing a [Privacy Scope](./RFC-PRIV.md#privacy-scope) Restriction, set the new **Restriction Scope** to be the intersection of the previous **Restriction Scope** with the Privacy Scope of the Restriction.
+        **Concerned Fragments** is set to all Data Capture Fragments the `scope`s of which are included in the **Restriction Scope**.
```

Comment:
Fragment whose scope contains at least 1 triple which is contained in the **Restriction Scope**?

Reply:
yes, [after both scopes have been resolved to all of their equivalent scopes.](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/expected-behavior.md#privacy-scope-triples)